code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -575,4 +575,12 @@
 		else if(can_be_inserted(target))
 			handle_item_insertion(target)
 			return 1
-	return 0
\ No newline at end of file
+	return 0
+
+/obj/item/weapon/storage/OnMobDeath(mob/wearer as mob)
+	for(var/obj/item/device/gps/secure/S in contents)

```

Comment:
Terrible, terrible hardcode.
Also I don't think you should do this, if someone wants the SPS protection they should carry it on them, not in a bag.